admin/pkg/admin/routes.go

```diff
@@ -15,15 +15,18 @@
 package admin
 
 import (
+	"fmt"
 	"net/http"
 )
 
 func (a *Admin) adminRoutes() *http.ServeMux {
 
 	adminMux := http.NewServeMux()
 
-	adminMux.HandleFunc("/v1/<something>/", func(writer http.ResponseWriter, request *http.Request) {
-		// Queries something
+	adminMux.HandleFunc("/admin/hellowworld", func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
There appears to be a typo here.
```suggestion
	adminMux.HandleFunc("/admin/helloworld", func(writer http.ResponseWriter, request *http.Request) {
```